src/main/java/pokecube/adventures/PokecubeAdv.java

```diff
@@ -327,6 +327,6 @@ public PokecubeAdv()
      */
     public void serverStarting(final FMLServerStartingEvent event)
     {
-        CommandConfigs.register(PokecubeAdv.config, event.getCommandDispatcher(), "pokeadvsettings");
+        CommandConfigs.register(PokecubeAdv.config, event.getResult(), "pokeadvsettings");
```

Comment:
Don't make this fix yet, the entire required event has changed, so it will need some more changes elsewhere instead, same for anything else trying to deal with commands from FMLServerStartingEvent